packs/feats/dual-weapon-reload.json

```diff
@@ -17,7 +17,11 @@
             "value": 1
         },
         "prerequisites": {
-            "value": []
+            "value": [
```

Comment:
This is a Gunslinger feat.

Reply:
Ah!

It got reprinted in PC2 as part of the dedication.

Fun times - do we need two versions of this feat?

Reply:
I defer to @TikaelSol.

Reply:
Wait

We had this same issue before. It was resolved by doing something like `Gunslinger or Dual Weapon Dedication`

Check Douse Spell

Reply:
I remember talking about it, but don't know that Dousing Spell was the one that got the weird prereq stuff. I think technically this one may need to just get listed as an additional feat for the archetype? Assuming it is reprinted in the remaster Guns and Gears as is.

Reply:
I moved the dedication to the additional feats section for the moment. We can revisit this once Guns and Gears remaster hits.